lib/lotus/helpers/numbers/number_formatter.rb

```diff
@@ -0,0 +1,72 @@
+module Lotus
+  module Helpers
+    module Numbers
+      # Numbers
+      #
+      # @since x.x.x
+      class NumberFormatter
+        # NumberFormatter
+        #
+        # @since x.x.x
+        # @api private
+
+        def parse
+          parts.join(separator)
+        end
+
+        private
```

Comment:
@AlfonsoUceda I made these methods private as they are only used internally. Why do you think it should be removed?